src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1631,8 +1631,9 @@
    64 rededuplicate-facts
    65 varchar-columns-to-text
    66 jsonb-facts
-   67 add-resource-events-pk
-   68 support-fact-expiration-configuration})
+   ;; migration 69 replaces migration 67 - we do not need to apply both
```

Comment:
Not sure, but wonder if it'd be worth a little more detail here, perhaps mentioning why it's ok to ignore the previous one, i.e. because this one just rewrites the same table in a slightly different way, or...